PSModule/Helper.cs

```diff
@@ -364,10 +364,10 @@ public static int GetNumberOfTests(IList<ReportMetaData> listReport, out IDictio
                 table.RenderControl(new HtmlTextWriter(sw));
                 html = sw.ToString();
             }
-            File.WriteAllText($@"{uftWorkingFolder}\res\Report_{buildNumber}\Run Summary", html);
+            File.WriteAllText($@"{rptPath}\{RUN_SUMMARY}", html);

```

Comment:
It is recommended to use `Path.Combine` method instead of concatenating the path by string.